recirq/otoc/loschmidt/tilted_square_lattice/analysis-walkthrough.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loschmidt Analysis Walkthrough"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "import matplotlib\n",
+    "matplotlib.rcParams.update(**{\n",
+    "    'axes.titlesize': 14,\n",
+    "    'axes.labelsize': 14,\n",
+    "    'xtick.labelsize': 12,\n",
+    "    'ytick.labelsize': 12,\n",
+    "    'legend.fontsize': 12,\n",
+    "    'legend.title_fontsize': 12,\n",
+    "    'figure.figsize': (7, 5),\n",
```

Comment:
What's up with these magic values?  Maybe a comment to explain?

Reply:
added

```
# Set up reasonable defaults for figure fonts
```